CDCgov/prime-simplereport
# Remove and refactor reference to result column

# FRONTEND PULL REQUEST

## Related Issue
#4222 

## Changes Proposed
- Remove the reference to the `result` column in the front end so that we can remove references to it in the back end

## Additional Information
N/A

## Testing
1. Go to the "Results" tab of an org
2. Ensure multiplex and covid results are displaying as intended
 
## Screenshots / Demos
<img width="1345" alt="Screen Shot 2022-09-14 at 8 15 28 PM" src="https://user-images.githubusercontent.com/20211771/190290515-b657907e-a104-497b-80e7-78975fdc70d3.png">

## Checklist for Author and Reviewer

### Design

### Content

### Support

### Security

### Documentation
